File: java/console/src/main/java/com/lacunasoftware/signer/sample/scenarios/NotifyFlowParticipantsScenario.java
package com.lacunasoftware.signer.sample.scenarios;

import java.io.IOException;

import com.lacunasoftware.signer.documents.CreateDocumentResult;
import com.lacunasoftware.signer.documents.DocumentModel;
import com.lacunasoftware.signer.flowactions.FlowActionModel;
import com.lacunasoftware.signer.reserveds.RestException;

@Deprecated
public class NotifyFlowParticipantsScenario extends Scenario {
    /**
    * This scenario demonstrates how to notify participants 
    * of the flow.
    */
    @Override
    public void Run() throws IOException, RestException, Exception {
        // 1. Get a document Id
        CreateDocumentResult result = createDocument();

        // 2. Get the document details
        DocumentModel details = signerClient.getDocumentDetails(result.getDocumentId());

        // 3. Notify each participant individually if necessary
        //    Note: Only participants with pending actions are notified.
        for (FlowActionModel flowAction : details.getFlowActions()) {
            signerClient.sendFlowActionReminder(result.getDocumentId(), flowAction.getId());
        }
    }
}

File: php/Scenarios/NotifyFlowParticipantsScenario.php
<?php


namespace Lacuna\Scenarios;


use Lacuna\Signer\Model\DocumentsDocumentModel;
use Lacuna\Signer\Model\FlowActionsFlowActionModel;

class NotifyFlowParticipantsScenario extends Scenario
{
    /**
     * This scenario demonstrates how to notify participants
     * of the flow.
     */
    function run()
    {
        // 1. Get a document Id
        $result = $this->createDocument();

        // 2. Get the document details
        $details = new DocumentsDocumentModel($this->signerClient->getDocumentDetails($result->getDocumentId()));

        // 3. Notify each participant individually if necessary
        //    Note: Only participants with pending actions are notified.
        foreach ($details->getFlowActions() as $item) {
            $flowAction = new FlowActionsFlowActionModel($item);
            var_dump($flowAction);
            $this->signerClient->sendFlowActionReminder($result->getDocumentId(), $flowAction->getId());
        }

    }

}